### ISSUE TITLE
With a fresh install and Laravel Herd, Blast doesn't work

### ISSUE THREAD

**Issue by domthomas-dev** [area17/blast#116]
Hi,

I install a fresh Laravel 10 with Laravel Herd. I install this package, and I try to launch it, but there are much errors in terminal 
I followed the doc, twice, but without result.
```
[storybook] info => Serving static files from ./../../../public at /
[storybook] info => Starting manager..
[storybook] WARN unable to find package.json for @prettier/plugin-php
[storybook] WARN unable to find package.json for eslint
[storybook] WARN unable to find package.json for eslint-config-prettier
WARN unable to find package.json for eslint-plugin-prettier
[storybook] WARN unable to find package.json for husky
[storybook] WARN unable to find package.json for lint-staged
WARN unable to find package.json for prettier
[storybook] WARN unable to find package.json for stylelint
[storybook] WARN unable to find package.json for stylelint-config-recommended
[storybook] WARN unable to find package.json for stylelint-order
[storybook] WARN unable to find package.json for stylelint-prettier
[storybook] info Addon-docs: using MDX2
[storybook] info => Using implicit CSS loaders
[storybook] info => Using default Webpack5 setup
```

And when I start the website : 
`Too few arguments to function A17\Blast\Controllers\StoryController::__invoke(), 1 passed`

When I launch the `button` demo, I have this : 
Undefined variable $href

**Comment by mrtimbrook** [comment]
After starting blast are you going to http://localhost:6006?

**Comment by domthomas-dev** [comment]
Indeed .... I'm sorry.

So, now, I have this
![Screenshot 2024-03-11 at 11 12 13](https://github.com/area17/blast/assets/17202290/0a6d5c97-12a4-434b-8541-09b121d2341b)

**Comment by mrtimbrook** [comment]
If you open the network tab in your dev tools you'll see the request storybook is making to your application. Open that url in a new tab you should see the full error causing the fetch error

**Comment by domthomas-dev** [comment]
It's a CORS problem
My Storybook have an URL in *.test

**Comment by domthomas-dev** [comment]
I will use Laravel 11 to do this. 
When do you think is it available ?

**Comment by mrtimbrook** [comment]
@domthomas-dev I published blast 1.10.0 last night which adds Laravel 11 support

**Comment by domthomas-dev** [comment]
I'm going to test it the night. I think I need this storybook for my project :)
And I'm going to resume with a fresh install :)
